Applications/Terminal/TerminalView.m

```diff
@@ -2967,6 +2967,15 @@ - (void)setAlternateAsMeta:(BOOL)altAsMeta
 // ---
 // Title (window, icon)
 // ---
+- (NSString *)windowTitle
+{
+  return title_window;
+}
```

Comment:
I think it's better to name it `xtermTitle` to reflect its value and use it along with other similar methods: `-programPath`, `-deviceName` and `-windowSize` to construct title in TerminalWindow's `-updateTitleBar`.